docs/Introduction.md

```diff
@@ -0,0 +1,122 @@
+### Data pipelines for everyone with CRMint
+#### Automate your BigQuery SQL queries with this elegant GCP pipeline application.
+
+![](https://cdn-images-1.medium.com/max/800/1*QSxSRgKOzt39wrY19Fa1hg.jpeg)
```

Comment:
Could we perhaps move the images to our own documentation, in preference to relying on medium.com?

Perhaps under `docsource/img/`